.github/workflows/main.yml

```diff
@@ -1,18 +1,32 @@
-name: deploy development branch to linchpino-core-dev server
+name: Build and Deploy development branch to linchpino-core-dev server
 on:
-  workflow_run:
-    workflows: [ "SonarCloudMaven" ]
-    types:
-      - completed
   push:
- 
+    branches:
+      - '**'
 env:
   REGISTRY: ghcr.io
   IMAGE_NAME: ${{ github.repository }}
-
 jobs:
+  sonar-build:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/development'
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up JDK 18
```

Comment:
We are using Java 21. Is it possible to upgrade it?

Reply:
Dear Mehdi 
done